app/src/main/java/com/example/hadar/trempyteam/searchTrempFragment.java

```diff
@@ -104,17 +102,15 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                         String contry = listAddresses.get(0).getLocality();
```

Comment:
IMO, `listAddresses.get(0)` should be extracted into a variable to avoid list-access `get()` method.